app/services/analytics_events.rb

```diff
@@ -4639,14 +4639,14 @@ def reactivate_account_visit
   # @param [Hash] recaptcha_result Full reCAPTCHA response body
   # @param [Float] score_threshold Minimum value for considering passing result
   # @param [Boolean] evaluated_as_valid Whether result was considered valid
-  # @param [String] validator_class Class name of validator
+  # @param [String] form_class Class name of form
```

Comment:
I'd consider leaving this so it's easier to track over time?

Reply:
Normally I'd agree if we were already using this in dashboards or saved queries, but I think this was added primarily as a way to diagnose individual log entries, so I don't know that I'm quite as concerned about backwards compatibility?

Reply:
makes sense then